docs/make.jl

```diff
@@ -4,18 +4,42 @@ using Literate
 
 # generate examples
 for example in ["cholesky", "blur-roberts"]
-    dir = joinpath(DataFlowTasks.PROJECT_ROOT, "docs", "src", "examples", example)
-    src = joinpath(dir, "$(example).jl")
-    Literate.markdown(src, dir)
-    Literate.notebook(src, dir)
+    println("\n*** Generating $example example")
+    @time begin
+        dir = joinpath(DataFlowTasks.PROJECT_ROOT, "docs", "src", "examples", example)
+        src = joinpath(dir, "$(example).jl")
+        Literate.markdown(src, dir)
+        Literate.notebook(src, dir)
+    end
 end
 
 # generate readme
-dir = joinpath(DataFlowTasks.PROJECT_ROOT, "docs", "readme")
-src = joinpath(dir, "README.jl")
-Literate.markdown(src, dir, flavor=Literate.CommonMarkFlavor())
-Literate.notebook(src, dir)
+println("\n*** Generating README")
+@time cd(joinpath(DataFlowTasks.PROJECT_ROOT, "docs", "src", "readme")) do
+    src = joinpath(pwd(), "README.jl")
 
+    # Run code
+    include(src)
+
+    # Generate notebook
+    Literate.notebook(src, pwd())
+
+    # Generate markdown
+    # -> fix image paths to link to github.io
+    Literate.markdown(src, pwd(), flavor=Literate.CommonMarkFlavor())
+    contents = read("README.md", String)
+    contents = replace(contents, "![](" => "![](https://maltezfaria.github.io/DataFlowTasks.jl/dev/readme/")
+    write(joinpath("README.md"), contents)
```

Comment:
Why is `joinpath` needed?

Reply:
`joinpath` isn't useful. But reviews are!